Vagrantfile

```diff
@@ -112,6 +113,15 @@ Vagrant.configure("2") do |global_config|
         end
       end
 
+      config.vm.provider "libvirt" do |lv, override|
```

Comment:
Does `lv` have an equivalent of `vb.gui`? Eh, I suppose we could add it later if/when needed...